libr/io/io_desc.c

```diff
@@ -1,10 +1,11 @@
-/* radare2 - LGPL - Copyright 2017-2019 - condret, pancake, alvaro */
+/* radare2 - LGPL - Copyright 2017-2020 - condret, pancake, alvaro */
 
 #include <r_io.h>
 #include <sdb.h>
 #include <string.h>
 
-//shall be used by plugins for creating descs
+// shall be used by plugins for creating descs
+// calling this function many times result in always fd=0
```

Comment:
why?

Reply:
just read the code, is quite obvious, in short:

* fd32 is only updated when io->files != NULL
* this function doersnt call or set anything to change io->files
* idpool_grab_id is never called

but again, this is far from the scope of this PR and the discussion can go on and on for years, i dont think it should be addressed in here

Reply:
i was speaking from the unit-test pov, because this io->files chk seems wrong to me. but as i said, i think this should be discussed and addressed in a different place. not in this pr

Reply:
I don't see that as a problem in unit tests either.

Anyway, I agree to not discuss this in this PR, but you can't add *random* comments and then say "this is not the place to discuss it". If it's not the place to discuss it, just don't add the comment in the first place.